### DISCUSSION TITLE
How can I regist backquote as thing?

### DISCUSSION THREAD

**Discussion by lakkiy** [meow-edit/meow#525] [Q&A] [answered]
I was take a look at function's doc and source code and write this but it didn't work:

```elisp
(meow-thing-register 'backquote
                     '(pair ("`") ("`"))
                     '(pair ("`") ("`")))
(add-to-list 'meow-char-thing-table '(?` . backquote))
```

PS: I want use meow-inner/bound-of-thing to select string in backquote in golang, for the first time I think meow-inner-of string should be work but it not, so I try to regist backquote as thing.

**Comment by eshrh** [comment, marked as answer]
I'm currently unsure why this doesn't work, but using regexps seems to work fine.

**Reply by lakkiy** [reply]
Thanks, it  works with regexp.
